Makefile

```diff
@@ -15,27 +15,15 @@ install:
 concourse_e2e:
 	@echo "Executing e2e automated tests against the staging environment..."
 	# execute all scenarios except no submission and backend submissions test ( the '-' character prefix means skip)
-ifeq (${TIERING_LOGIC},True)
```

Comment:
what if we have to revert the feature flag e.g set it to false the the e2e will fail?

Reply:
Will we ever need to go back? I think at this point the tiering stuff has been in production long enough that the feature flagging should be removed. Am I wrong about that?

Reply:
Its a good question i am not sure to be honest my understanding was to keep it so if it did go into a national lockdown it could easily be toggled back but this view of the world may be outdated now